### PR TITLE
Initial CaaSPv4 installation workflow (fate#325834)

### PR DESCRIPTION
**Pull request by sysrich**

### CODE REVIEW

**Review by imobachgs — APPROVED** [review]

File: control/control.CAASP.xml [hunk 1]
```diff
@@ -437,31 +354,78 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
...
+                </proposal_module>
+                <proposal_module>
+                    <name>hwinfo</name>
+                    <presentation_order>80</presentation_order>
+                </proposal_module>
+                <proposal_module>
+                    <name>partitions</name>
+                    <presentation_order>10</presentation_order>
+                </proposal_module>
+                <!-- software proposal should be computed almost at the end -->
```
Comment (jreidinger) [inline control/control.CAASP.xml]: I worry that this comment and reality is not in sync. But software should be definitively after timezone, network and kdump as all of them may need to add to software proposal additional software. ( if it does not do that now, it is bug and we will fix it ).

Comment (sysrich) [reply control/control.CAASP.xml]: @thkukuk what do you think? You're the one who wanted software so high on the list

we haven't seen a problem with this on Kubic but I see exactly where @jreidinger is coming from

Comment (jreidinger) [reply control/control.CAASP.xml]: well, you see mainly difference when you play with proposal later, then some software which is required can miss. And also please note again that it is not order which is used for displaying it ( presentation order is used ), only order in which proposals are computed. And other modules can say software proposal that it need something and that software should add it.

**Review by jreidinger — APPROVED** [review]

File: control/control.CAASP.xml (hunk 1 above)
Comment (thkukuk) [reply control/control.CAASP.xml]: As Josef wrote, display order is not compute order. Compute the software proposal as last step sounds correct to me, too.

**Pull request merged**